.travis.yml

```diff
@@ -7,11 +7,23 @@ go:
   - "1.11"
   - master
 
+env:
+  global:
+    - REVIEW_DOG_ON=1.11
+    - REVIEW_DOG_VERSION=0.9.11
+
 branches:
   only:
     - master
 
 install:
+  - |
+    if [ ${TRAVIS_GO_VERSION} == ${REVIEW_DOG_ON} ]; then
+      mkdir -p ${GOPATH}/bin
+      go get -u golang.org/x/lint/golint
```

Comment:
go modules is not yet enabled in this repo and golint is not provided as binary release.
let's support go modules later.